### PR TITLE
KV Store: Database migration docs

### PR DESCRIPTION
**Pull request by N-o-Z**

### CODE REVIEW

**Review by nopcoder — CHANGES_REQUESTED** [review]
Fill content

File: docs/reference/database-migration.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+has_children: false
+---
+
+# lakeFS Database Migrate
+{: .no_toc }
+
+**Note:** Feature in development
+{: .note }
+
+lakeFS database migration allows users to easily transition from one database implementation to another.
```
Comment (nopcoder) [inline docs/reference/database-migration.md:16]: Maybe:
The lakeFS database migration tool makes it easy for users to switch from one database implementation to another.

**Review by nopcoder — APPROVED** [review]
Small suggestion

File: docs/reference/database-migration.md (hunk 1 above)
Comment (N-o-Z) [reply docs/reference/database-migration.md]: @nopcoder See updated upgrade comment

**Pull request merged**